3-extra/2-sorting-algorithm.js

```diff
@@ -14,7 +14,15 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {}
+function sortAges(arr) {
+  let nonNumberArr = []
+  for (let item of arr){
+      if (typeof item==='number'){
+        nonNumberArr.push(item)
+      }
+  }
```

Comment:
when you create a new array based on a condition you can always use `filter` instead

Reply:
in task they asked not to use filter :)
removes any non-number data types without using the built-in javascript filter method